components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -65,35 +68,44 @@
     /**
      * Returning instance of one of the implementation of SessionDataStore.
      *
-     * @return {@link RdbmsSingleEntrySessionDataStore} or {@link RedisSessionDataStore}
+     * @return {@link RdbmsSingleEntrySessionDataStore} or {@link RedisSessionDataStore} or {@link RdbmsMultiEntrySessionDataStore}
```

Comment:
Lets get rid of implementation based information therefore put a general return statement in this java doc